site/using-weave/service-management.md

```diff
@@ -10,6 +10,7 @@ This section contains the following topics:
  * [Binding Services](#binding)
  * [Routing Services](#routing)
  * [Dynamically Changing Service Locations](#change-location)
+ * [Exposing the Weave Network More Widely](#export-route)
```

Comment:
Looking at the current doc structure, I don't think this is the right place for this. It is not about services and instead is more like the "Integrating with the Host Network" section which lives at the same level as this entire doc. So I reckon it should either go into that (and that section should be renamed) or be a separate section at the same level.

@abuehrle what do you think?